helm/defectdojo/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "2.2.0-dev"
 description: A Helm chart for Kubernetes to install DefectDojo
 name: defectdojo
-version: 1.6.11-dev
+version: 1.6.12-dev
```

Comment:
@madchap Why is the version changed to 1.6.12-dev? The current version in master is 1.6.10, in dev we changed to 1.6.11-dev, so I assumed the next master will have version 1.6.11.

Reply:
Not sure if this was changed at some point, but I think the linter expects a version bump every time something changes in the chart.

Reply:
I changed it into 1.6.11-dev, let's see what happens :-)

Reply:
✖︎ defectdojo => (version: "1.6.11-dev", path: "helm/defectdojo") > Chart version not ok. Needs a version bump!

Reply:
going to set it to 1.6.12-dev and merge it for the release. someone who knows helm should figure out this linting stuff as it seems to restrive atm